R/label_roche.R

```diff
@@ -164,6 +164,7 @@ label_roche_ratio <- function(digits = 2,
 }
 
 #' @param inf character value used to style infinite values in the table.
+#' @param nan character value used to style NaN values in the table.
```

Comment:
Can you move these two lines to below the `@inheritParams` calls?

Also, can you style them like the `na` argument and include the default value?

Reply:
@ayogasekaram please be sure to respond to comments when you've made updates based on the comment, thanks!